include/dem/pw_linear_force.h

```diff
@@ -83,7 +83,8 @@ class PWLinearForce : public PWContactForce<dim>
       &           pw_pairs_in_contact,
     const double &dt,
     std::unordered_map<types::particle_index, Tensor<1, dim>> &momentum,
-    std::unordered_map<types::particle_index, Tensor<1, dim>> &force) override;
+    std::unordered_map<types::particle_index, Tensor<1, dim>> &force,
+    const DEMSolverParameters<dim> &dem_parameters) override;
```

Comment:
Why are the DEM solver parameters needed here?

Reply:
I need a bunch of parameters in dem_parameters.